shared/util/versions.ts

```diff
@@ -33,8 +33,10 @@ export function compareVersion({
         return enableForDev
     }
 
-    if (semver.valid(productVersion)) {
-        return semver.satisfies(productVersion, `>=${minimumVersion}`)
+    // Split any -rc.x tags from the end of the product version
+    const semanticVersion = productVersion.split('-')[0]
```

Comment:
We might not use it, but `1.2.3+blah` is also grammatically legal (as is `1.2.3-foo+blah`).

Reply:
Updated to be a little more general.